src/main/java/org/stellar/sdk/FeeBumpTransaction.java

```diff
@@ -202,4 +201,21 @@ public FeeBumpTransaction build() {
           this.accountConverter, this.feeAccount, this.baseFee, this.innerTransaction);
     }
   }
+
+  @Override
+  public boolean equals(Object object) {
+    if (this == object) {
+      return true;
+    }
+    if (object == null || getClass() != object.getClass()) {
+      return false;
+    }
+    FeeBumpTransaction that = (FeeBumpTransaction) object;
+    return Objects.equals(toEnvelopeXdr(), that.toEnvelopeXdr());
```

Comment:
how about using the `hash()` method from AbstractTransaction instead of `toEnvelopeXdr()` it encapsulates same goal of serialization of tx envelope, but also compiles in the network passphrase to the hash.